src/main.js

```diff
@@ -58,5 +60,16 @@ new Vue({
 	el: '#content',
 	router,
 	store,
+	mounted: function() {
+		this.$OC.Search = new OCA.Search(this.filter, this.cleanSearch)
```

Comment:
no need to assing it anywhere for now, just call `new OCA.Search()`

Reply:
If I change 
this.$OC.Search = new OCA.Search(this.filter, this.cleanSearch)
to
this.$OC.Search = new OCA.Search()
the search field disappears.

Reply:
Sorry, you got me wrong there. You can just remove the assignment to `this.$OC.Search` and call `new OCA.Search(this.filter, this.cleanSearch)` without assigning the result.